OneLocBuild is failing in official builds

https://dnceng.visualstudio.com/internal/_build/results?buildId=2089986&view=logs&j=6aa31087-d1b8-5ee1-af29-f0fe63353381&t=f5f0ecb8-e09a-5b29-99cb-311e048f2c1c

Will disable the leg meanwhile to unblock official builds.

```
OneLocBuildClient.exe Information: 0 : Uploading '104' localized files in payload size '2960500' bytes...
OneLocBuildClient.exe Error: 0 : Octokit.ApiException: Sorry, your request timed out.  It's likely that your input was too large to process.  Consider building the tree incrementally, or building the commits you need in a local clone of the repository and then pushing them to GitHub.
   at Octokit.Connection.HandleErrors(IResponse response) in /home/runner/work/octokit.net/octokit.net/Octokit/Http/Connection.cs:line 709
   at Octokit.Connection.<RunRequest>d__61.MoveNext() in /home/runner/work/octokit.net/octokit.net/Octokit/Http/Connection.cs:line 686
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Octokit.Connection.<Run>d__60`1.MoveNext() in /home/runner/work/octokit.net/octokit.net/Octokit/Http/Connection.cs:line 670
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Octokit.ApiConnection.<Post>d__23`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Polly.Retry.AsyncRetryEngine.<ImplementationAsync>d__0`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Polly.AsyncPolicy.<ExecuteAsync>d__21`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at OneLocBuildClient.GitHubCheckin.<CallGitHubApiWithRetryAsync>d__24`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at OneLocBuildClient.GitHubCheckin.<PushTreeItemsAsync>d__20.MoveNext() in C:\__w\1\s\src\OneLocBuildClient\GitHubCheckin.cs:line 387
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at OneLocBuildClient.GitHubCheckin.<PushLocFilesAsync>d__19.MoveNext() in C:\__w\1\s\src\OneLocBuildClient\GitHubCheckin.cs:line 363
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at OneLocBuildClient.GitHubCheckin.<RunAsync>d__12.MoveNext() in C:\__w\1\s\src\OneLocBuildClient\GitHubCheckin.cs:line 133
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at OneLocBuildClient.LocController.<RunAsync>d__6.MoveNext() in C:\__w\1\s\src\OneLocBuildClient\LocController.cs:line 67
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at OneLocBuildClient.Program.Main(String[] args) in C:\__w\1\s\src\OneLocBuildClient\Program.cs:line 46
{"message":"Sorry, your request timed out.  It's likely that your input was too large to process.  Consider building the tree incrementally, or building the commits you need in a local clone of the repository and then pushing them to GitHub.","documentation_url":"[https://docs.github.com/rest/reference/git#create-a-tree"}](https://docs.github.com/rest/reference/git#create-a-tree%22})
##[error]Process 'OneLocBuildClient.exe' exited with code '1'.
Finishing: OneLocBuild
```

--- a/eng/pipelines/runtime-official.yml
+++ b/eng/pipelines/runtime-official.yml
@@ -39,14 +39,12 @@ extends:
       # Localization build
       #
 
-      # TODO: Re-enable when https://github.com/dotnet/runtime/issues/80729 is fixed.
-
-      #- ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/main') }}:
-      #  - template: /eng/common/templates/job/onelocbuild.yml
-      #    parameters:
-      #      MirrorRepo: runtime
-      #      LclSource: lclFilesfromPackage
-      #      LclPackageId: 'LCL-JUNO-PROD-RUNTIME'
+      - ${{ if eq(variables['Build.SourceBranch'], 'refs/heads/main') }}:
+        - template: /eng/common/templates/job/onelocbuild.yml
+          parameters:
+            MirrorRepo: runtime
+            LclSource: lclFilesfromPackage
+            LclPackageId: 'LCL-JUNO-PROD-RUNTIME'
 
       #
       # Source Index Build

